### DISCUSSION TITLE
(mobile app) Provide a way to search repos within organization

### DISCUSSION THREAD

**Discussion by stevula** [community/community#4630] [Mobile] [answered]
My org has hundreds of private repos but when I browse them through the organization screen (Organization > Repositories) in mobile there is no search bar and I have to scroll through a huge unsorted list. I usually have to leave the app and search in the browser if I want to find a specific internal repo. 

Suggestion: add a search bar to the organization's repos screen in the mobile app.

**Comment by Skwesy-187** [comment]
> My org has hundreds of private repos but when I browse them through the organization screen (Organization > Repositories) in mobile there is no search bar and I have to scroll through a huge unsorted list. I usually have to leave the app and search in the browser if I want to find a specific internal repo. 
> 
> Suggestion: add a search bar to the organization's repos screen in the mobile app.

**Comment by mxie** [comment, marked as answer]
Thanks for your feedback @stevula! I will pass this along to the team for internal discussion.

Related discussion: https://github.com/github/feedback/discussions/4364
